internal/handlers/common/filter.go

```diff
@@ -974,31 +958,21 @@ func filterFieldExprBitsAnySet(fieldValue, maskValue any) (bool, error) {
 	}
 }
 
+// isInvalidBitwiseValue returns true for invalid values of float64
+// use for bitwise operation.
+// Non integer float64, Nan, Inf are unsupported.
+// The value must be less than math.MaxInt64 (exclusive),
+// and mst be greater than or equal to math.MinInt64 (inclusive).
+func isInvalidBitwiseValue(value float64) bool {
+	return value != math.Trunc(value) ||
+		math.IsNaN(value) ||
+		math.IsInf(value, 0) ||
+		value >= math.MaxInt64 ||
```

Comment:
See this logic changed from `value > math.MaxInt64` to `value >= math.MaxInt64`. This is because `test-integration-mongodb` said so for newly added Scalars dataset.

Reply:
Okay, so from my understanding here - MongoDB doesn't support values bigger or equal to MaxInt64

Reply:
It appears so. But I didn't find them in any documentation so this change is purely based on test result of `test-integration-mongodb` :hugs: 

And this is what I've fixed from moving _overflow_ and _verge_ to Scalars dataset.